ops/terraform/modules/stateful/main.tf

```diff
@@ -242,6 +242,11 @@ resource "aws_db_parameter_group" "import_mode" {
   parameter {
     name  = "autovacuum"
     value = "0"
+
```

Comment:
This looks invalid. Didn't you goof a closing brace here?

Reply:
I did, thanks for catching that, it has been fixed.